### DISCUSSION TITLE
Issue while installing Stackstorm  on the URL https://index.stackstorm.org/v1/index.json

### DISCUSSION THREAD

**Discussion by ArulananthamParamasivam** [StackStorm/st2#5416] [Operations]
Hi Team,

While installing the Stackstorm application on Linux RHEL, we got the below error.

id: 616e6a699a9a1e197488f1c3
action.ref: packs.download
context.user: st2admin
parameters:
  dependency_list: null
  force: false
  packs:
  - st2
status: failed (1s elapsed)
start_timestamp: Tue, 19 Oct 2021 06:49:13 UTC
end_timestamp: Tue, 19 Oct 2021 06:49:14 UTC
log:
  - status: requested
    timestamp: '2021-10-19T06:49:13.597000Z'
  - status: scheduled
    timestamp: '2021-10-19T06:49:13.661000Z'
  - status: running
    timestamp: '2021-10-19T06:49:13.722000Z'
  - status: failed
    timestamp: '2021-10-19T06:49:14.406000Z'
result:
  exit_code: 1
  result: None
  stderr: "Index parsing error: {
  "url": "https://index.stackstorm.org/v1/index.json",
  "packs": 0,
  "message": "SSLError(MaxRetryError(\\"HTTPSConnectionPool(host='index.stackstorm.org', port=443): Max retries exceeded with url: /v1/index.json (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:897)'),))\\",),)",
  "error": "unresponsive"
}
Traceback (most recent call last):
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/python_runner/python_action_wrapper.py", line 395, in <module>
    obj.run()
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/python_runner/python_action_wrapper.py", line 214, in run
    output = action.run(**self._parameters)
  File "/opt/stackstorm/packs/packs/actions/pack_mgmt/download.py", line 106, in run
    return self._validate_result(result=result, repo_url=pack_url)
  File "/opt/stackstorm/packs/packs/actions/pack_mgmt/download.py", line 129, in _validate_result
    raise Exception(message)
Exception: The pack has not been downloaded from "None".

Errors:
No results from the index: tried https://index.stackstorm.org/v1/index.json.
Status: [
  {
    _"url": "https://index.stackstorm.org/v1/index.json",
    "packs": 0,
    "message": "SSLError(MaxRetryError(\\"HTTPSConnectionPool(host='index.stackstorm.org', port=443): Max retries exceeded with 
	url: /v1/index.json (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:897)'),))\\",),)",
    "error": "unresponsive"_
  }
]
"
  stdout: ''

**Comment by minsis** [comment]
You're either not able to connect outbound or your ca-certs are really old.

Can you `curl -v https://index.stackstorm.org/v1/index.json` from the same server?
